### ISSUE TITLE
This is python 3

### ISSUE THREAD

**Issue by fishbacon** [f-klubben/stregsystem#2]
Who uses new versions?!

**Comment by HenrikOssipoff** [comment]
Python 3 is awesome, and 2.x will lose support in 2020. Give that we likely wont be done until then, Python 3 seems like a good fit.

**Comment by HenrikOssipoff** [comment]
We might as well use this issue as a Q&A option for Python 2 vs. Python 3 methodologies. I've done quite a bit of work with Python 3 (and Python 2 vs. 3 compatability in libraries for that matter) so feel free to throw question at me for best practices.

**Comment by fishbacon** [comment]
I found out `reduce` was removed from built-ins which made me a bit sad.

**Comment by HenrikOssipoff** [comment]
True, still available from functools though, as far as I remember.
